From a68bad515e7e2780756b76d12f193f580d1e61f4 Mon Sep 17 00:00:00 2001
From: Dominique Leuenberger <dominique-gnomezilla@leuenberger.net>
Date: Tue, 2 Mar 2010 18:39:00 +0100
Subject: [PATCH] More GCC 4.5 fixes, #611588

	* src/langs/nmv-cpp-ast.cc (Token::operator=): Token::Token is
	the name of the constructor, not of the type.
---
 src/langs/nmv-cpp-ast.cc |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/langs/nmv-cpp-ast.cc b/src/langs/nmv-cpp-ast.cc
index fbbe559..9e17d1b 100644
--- a/src/langs/nmv-cpp-ast.cc
+++ b/src/langs/nmv-cpp-ast.cc
@@ -64,7 +64,7 @@ Token::Token (const Token &a_t)
     m_int_value = a_t.get_int_value ();
 }
 
-Token::Token&
+Token&
 Token::operator= (const Token &a_t)
 {
     m_kind = a_t.get_kind ();
-- 
1.7.2.3

